src/Profiles.ts

```diff
@@ -305,6 +305,32 @@ export class Profiles { // Processing stops if there are no profiles detected
         return [updSession.ISession.user, updSession.ISession.password, updSession.ISession.base64EncodedAuth];
```

Comment:
Also is where is the invalid error being returned :/ in case of invalid credential entry

Reply:
Invalid error can be returned anywhere. #388 is an open issue that consolidates this error message. I am partially working on this in handle-invalid-credentials branch